scripting/include/smlib/sql.inc

```diff
@@ -19,14 +19,14 @@
  * @param ...			Variable number of format parameters.
  * @noreturn
  */
-stock SQL_TQueryF(Handle:database, SQLTCallback:callback, any:data, DBPriority:priority=DBPrio_Normal, const String:format[], any:...) {
+stock void SQL_TQueryF(Handle database, SQLTCallback callback, any data, DBPriority priority=DBPrio_Normal, const char[] format, any ...) {

```

Comment:
Not sure about this one. `Database` methodmap and `SQLQueryCallback` typedef instead of `SQLTCallback`? Would break backwards compat. Leave as is for now.